### ISSUE TITLE
Cannot proceed past splash screen

### ISSUE THREAD

**Issue by DennoTheMenno** [osmandapp/OsmAnd#18474]
### Description

Sometimes I cannot proceed past Splash Screen, system hangs until Close App/Wait/Crash Report screen appears.

### Steps to reproduce

Open app

### Actual result

System hangs on Splash Screen until Close App/Wait/Crash Report screen appears.

Workaround is as soon as I notice that loading is taking longer than it should, I can Force Close the app and then open it again & it loads correctly as expected.
This workaround is successful 100% of the time.

### Expected result

App to load as normal

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version: 4.6.4 EDIT: Now on 4.6.5 and issue persists
Android/iOS version: Android 13
Device model: Samsung Galaxy S20FE
Crash-Logs: ?
```
I have submitted Crash Logs twice already, and received an email to report the issue here.

**Comment by scaidermern** [comment]
I experience the same problem from time to time. Closing OsmAnd and reopening it usually solves the problem.

**Comment by DmitryAlexei** [comment]
Duplicate of https://github.com/osmandapp/OsmAnd/issues/18302
This issue mainly occurs on Samsung devices
